.github/workflows/clangtidy_check.yml

```diff
@@ -22,6 +22,9 @@ jobs:
         
       - name: "Install dependencies"
         run: |
+          wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key 2>/dev/null | sudo apt-key add -
+          sudo add-apt-repository 'deb http://apt.llvm.org/focal llvm-toolchain-focal-10 main' -y
+          sudo apt-get update -q
```

Comment:
Doesn't Ubuntu focal already have clang-10 in its repositories?

Reply:
It was failing with the error and I found a solution over [here](https://github.com/actions/virtual-environments/issues/1536#issuecomment-698537248)

Reply:
same with the msvc because the newest version of the windows 2019 toolset is 14.27 and clang-6.0 is removed from the image also which I had to edit the script to install it.

Reply:
I think they updated the image because of which these errors are triggering right now.

Reply:
Okay, then. It is fine to not depend upon the image vendor. They update it quite regularly (weekly)

Reply:
could you fix these errors in the main repo? otherwise, if someone commits it will give unnecessary errors.